lib/parse-args.js

```diff
@@ -30,7 +32,14 @@ function buildYargs (withCommands = false) {
       alias: ['o', 'report-dir'],
       group: 'Reporting options',
       describe: 'directory where coverage reports will be output to',
-      default: './coverage'
+      default: defaultReportDir,
+      coerce: (opt) => {
+        if (isAbsolute(opt) === false) {
+          defaultReportDir = opt = resolve(process.cwd(), opt)
+        }
+
+        return opt
+      }
```

Comment:
These lines ensure absolute paths for the reports directory argument.  It makes use of the [function-level variable](https://github.com/bcoe/c8/pull/447/files?diff=unified&w=0#r1090096554) that could as be required by the temp-directory argument.